src/applications/vaos/vaos-entry.jsx

```diff
@@ -0,0 +1,11 @@
+import '../../platform/polyfills';
```

Comment:
I think we can remove the `../../` from the two platform imports. We have an alias for the platform folder in Webpack.